content/kubeone/master/architecture/requirements/infrastructure_management/_index.en.md

```diff
@@ -19,7 +22,7 @@ example Terraform configs can be found in the [KubeOne's GitHub repository][terr
 {{% notice note %}}
 The example Terraform configurations are optimized for the CI and might not
 be production-ready out of the box. We advise checking the
-[Production Recommendations]({{< ref "../../cheat_sheets/production_recommendations" >}})
+[Production Recommendations][production-recommendations]
```

Comment:
You can't use this syntax in the notice blocks. For some reason, it's not rendered properly. Instead, please leave it as it was before.